Dockerfile

```diff
@@ -22,17 +22,27 @@ FROM debian:${DEBIAN_VERSION}-slim as base
 ARG RUNDIR
 ARG ENTRYPOINT=/bin/bash
 ARG RUNTIME_PACKAGES
+ARG RUNTIME_TAR_PACKAGES
 
 RUN apt update -y && \
     apt install -y --no-install-recommends \
         libreadline8 \
         busybox \
         netcat-openbsd \
         procserv \
+        wget \
         $RUNTIME_PACKAGES && \
     apt clean && \
     rm -rf /var/lib/apt/lists/*
 
+RUN for url in $RUNTIME_TAR_PACKAGES; do \
+        download_dir=$(mktemp -d) && \
+        wget $url -P $download_dir && \
+        tar --no-same-owner -xvf $download_dir/* -C /; \
+    done && \
+    ldconfig && \
+    rm -rf /tmp/*
+
```

Comment:
As we discussed, this should be a separate patch (and it was going to use our tar helper, no?)

Also, `--no-same-owner` should be explained quickly in the commit.